hotModuleReplacement.js

```diff
@@ -40,7 +40,7 @@ function updateCss(el, url) {
   if (!url) {
     url = el.href.split('?')[0];
   }
-  if (el.isLoaded === false && url == el.href.split('?')[0]) {
+  if (el.isLoaded === false) {
```

Comment:
@aminland It this change ok for you ?

Reply:
yeah i don't know why i put that there in the first place lol